index-test.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <title>Fake huiswerk test</title>
+</head>
+<body>
+<h1>Dit is een github test</h1>
```

Comment:
En nog een leuke comment in je code erbij :)

Reply:
Top, dankjewel!